security/openconnect/src/opnsense/mvc/app/controllers/OPNsense/Openconnect/forms/general.xml

```diff
@@ -36,15 +36,45 @@
         <help>Select the type of hash. Possible values are SHA256 or SHA1.</help>
     </field>
     <field>
-        <id>general.group</id>
-        <label>Group Name</label>
+        <id>general.authgroup</id>
```

Comment:
I agree with @fabianfrz, this should stay as the old value. Its name only internal after all.

Reply:
I stand corrected. general.group is only in the development version so for me if @mimugmail wants to go with these more descriptive names you propose that's fine. Marking this resolved.